src/components/dialogs/network-modification/composite-modification/composite-modification-edition-dialog.tsx

```diff
@@ -0,0 +1,169 @@
+                ...computeLabel(modif),
+            }
+        );
+    };
+
+    const renderNetworkModificationsList = () => {
+        return (
+            <>
+                {modifications && (
+                    <List sx={unscrollableDialogStyles.scrollableContent}>
+                        {modifications.map((modification: NetworkModificationMetadata) => (
+                            <>
+                                <ListItem key={modification.uuid}>
+                                    <Box>{getModificationLabel(modification)}</Box>
+                                </ListItem>
+                                <Divider component="li" />
+                            </>
+                        ))}
+                    </List>
+                )}
+            </>
+        );
+    };
+
+    useEffect(() => {
+        setIsFetching(true);
+        getCompositeModificationContent(compositeModificationId)
+            .then((response) => {
+                if (response) {
+                    setModifications(response);
+                }
+            })
+            .catch((error) => {
+                snackError({
+                    messageTxt: error.message,
+                    headerId: 'cannotRetrieveCompositeModification',
+                });
+            })
+            .finally(() => setIsFetching(false));
+    }, [compositeModificationId, name, snackError]);
+
+    const closeAndClear = (event?: SyntheticEvent) => {
+        onClose(event);
+    };
+
+    const onSubmit = (formData: FormData) => {
+        saveCompositeModification(compositeModificationId, formData[FieldConstants.NAME])
+            .then(() => {
+                if (selectionForCopy.sourceItemUuid === compositeModificationId) {
+                    dispatch(setSelectionForCopy(NO_SELECTION_FOR_COPY));
+                    broadcastChannel.postMessage({
+                        NO_SELECTION_FOR_COPY,
+                    });
+                }
+                closeAndClear();
+            })
+            .catch((errorMessage) => {
+                snackError({
+                    messageTxt: errorMessage,
+                    headerId: 'contingencyListEditingError',
+                    headerValues: { name },
+                });
+            });
+    };
+
+    return (
+        <CustomMuiDialog
+            open={open}
+            onClose={closeAndClear}
+            titleId={titleId}
+            onSave={onSubmit}
+            removeOptional={true}
+            isDataFetching={isFetching}
+            language={languageLocal}
+            formSchema={schema}
+            formMethods={methods}
+            unscrollableFullHeight
+        >
+            {!isFetching && (
+                <Box sx={unscrollableDialogStyles.unscrollableContainer}>
+                    <CompositeModificationEditionForm />
+                    {renderNetworkModificationsList()}
+                </Box>
+            )}
+        </CustomMuiDialog>
+    );
+};
```

Comment:
export default CompositeModificationEditionDialog;

Reply:
I think that now we are supposed to do the export on the declaration line : `export const CompositeModificationEditionDialog`

Or did I misunderstand ?

Reply:
Nevermind. A recent massive commit from Tristan updated those rules. Check it out.

Reply:
Ok, perfect